packages/app-admin-users-cognito/src/plugins/installation.tsx

```diff
@@ -143,14 +142,22 @@ const Install: React.FC<InstallProps> = ({ onInstalled }) => {
                         </Grid>
                     </SimpleFormContent>
                     <SimpleFormFooter>
-                        <ButtonPrimary
-                            data-testid="install-security-button"
-                            onClick={ev => {
-                                submit(ev);
-                            }}
-                        >
-                            Create Admin User
-                        </ButtonPrimary>
+                        <Grid>
+                            <Cell span={8}>
+                                <p style={{ textAlign: "left" }}>By submitting the form, you agree to our Terms of Service and acknowledge our {privacyPolicyLink}.</p>
+                            </Cell>
+                            <Cell span={4}>
+                                <ButtonPrimary
+                                    data-testid="install-security-button"
+                                    onClick={ev => {
```

Comment:
Nitpick / while we're at it..

Let's change the `onClick` prop to:

```ts
onClick={() => {
    submit();
}}
```

Reply:
That `submit` actually **does** take an event as an input. It's not required, but it is supported. And with that, you can simplify down to `onClick={submit}`.